releasenotes/notes/fix-wrap-decorator-default-span-name-qualname-c2b3e30d7fe2996c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    core: use the qualified function name for the default span name in the
+    tracer wrap decorator.
```

Comment:
We should also have an `upgrade` section since this is a breaking change. 

Probably something like:

```
upgrade:
  - |
     core: qualified function name is now used by default for span names when using the
     `wrap` decorator. The old functionality can be kept by setting `config.fully_qualified_func_names = False`
````